src/Wallabag/CoreBundle/Controller/ConfigController.php

```diff
@@ -98,6 +100,24 @@ public function indexAction(Request $request)
             return $this->redirect($this->generateUrl('config'));
         }
 
+        // handle tagging rule
+        $taggingRule = new TaggingRule();
+        $newTaggingRule = $this->createForm(new TaggingRuleType(), $taggingRule);
```

Comment:
Related to the change to `form_start`, could you add a third parameter to specify the action url of the form?

```php
$newTaggingRule = $this->createForm(new TaggingRuleType(), $taggingRule, array('action' => $this->generateUrl('config').'#set5'));
```